### ISSUE TITLE
[Feature Request] Native Linux version

### ISSUE THREAD

**Issue by hydrohs** [SuperDude88/TWWHD-Randomizer#91]
**Describe the solution you'd like**
While the Windows desktop application works under Wine, for whatever reason the text rendering is very slightly wonky. Still usable, but native would be better.

**Describe alternatives you've considered**
I tried compiling myself on Arch Linux, but while the project built it immediately segfaults on launch

**Comment by SuperDude88** [comment]
Not really, unfortunately — I’m kinda out of my depth debugging crashes that deep in Qt (especially on Linux) and it still seems to be hitting the same segfault

**Comment by SuperDude88** [comment]
Update on this: the segfault on Linux was fixed with bb1a76fc90d02c474bbc7fba520b668dc76e6063 so it is now possible to get running if you build it yourself. I'm currently working on adding it to Actions so that builds can just be downloaded in the future.
